ezplatform/usr/local/share/ez/ez_functions.sh

```diff
@@ -27,8 +27,30 @@ function do_ez_web_server_web_directory_non_writable() {
   fi
 }
 
+function check_ez_database_exists() (
+   set +e
+   local DATABASE_EXISTS
+   if [ -n "$DATABASE_PASSWORD" ]; then
+     mysql -h"$DATABASE_HOST" -u"$DATABASE_USER" -p"$DATABASE_PASSWORD" "$DATABASE_NAME" -e "SHOW TABLES; SELECT FOUND_ROWS() > 0;" | grep -q 1
+     DATABASE_EXISTS=$?
+   else
+     mysql -h"$DATABASE_HOST" -u"$DATABASE_USER" "$DATABASE_NAME" -e "SHOW TABLES; SELECT FOUND_ROWS() > 0;" | grep -q 1
+     DATABASE_EXISTS=$?
+   fi
```

Comment:
this is susceptible to network failures or invalid credentials being considered as an empty database. There is already code in the PHP image that does similar but without interpreting errors as empty database and could instead be extracted for reuse https://github.com/continuouspipe/dockerfiles/blob/master/php/shared/usr/local/share/assets/assets_functions.sh#L90

Reply:
sorry, sam already said the same latter stuff, just I didn't click the link to see beforehand